docs/source/converting_tensorflow_models.rst

```diff
@@ -26,10 +26,10 @@ Here is an example of the conversion process for a pre-trained ``BERT-Base Uncas
 
    export BERT_BASE_DIR=/path/to/bert/uncased_L-12_H-768_A-12
 
-   transformers-cli convert --model_type bert \
-     --tf_checkpoint $BERT_BASE_DIR/bert_model.ckpt \
-     --config $BERT_BASE_DIR/bert_config.json \
-     --pytorch_dump_output $BERT_BASE_DIR/pytorch_model.bin
+   transformers-cli convert \
+     --tf_checkpoint_path $BERT_BASE_DIR/bert_model.ckpt \
+     --bert_config_file $BERT_BASE_DIR/bert_config.json \
+     --pytorch_dump_path $BERT_BASE_DIR/pytorch_model.bin
```

Comment:
I'm not sure these changes are correct?

- [tf_checkpoint](https://github.com/huggingface/transformers/blob/master/src/transformers/commands/convert.py#L32) seems to be correct
- [config](https://github.com/huggingface/transformers/blob/master/src/transformers/commands/convert.py#L37) seems to be correct
- [pytorch_dump_output](https://github.com/huggingface/transformers/blob/master/src/transformers/commands/convert.py#L35) seems to be correct

Are you sure you're not using an old version of transformers to verify?

Reply:
I verified on the master branch. I think you are looking into the wrong file:

https://github.com/huggingface/transformers/blob/8438bab38e1ea60efca181c92ebc7e4602f91848/src/transformers/convert_bert_original_tf_checkpoint_to_pytorch.py#L47

https://github.com/huggingface/transformers/blob/8438bab38e1ea60efca181c92ebc7e4602f91848/src/transformers/convert_bert_original_tf_checkpoint_to_pytorch.py#L50

https://github.com/huggingface/transformers/blob/8438bab38e1ea60efca181c92ebc7e4602f91848/src/transformers/convert_bert_original_tf_checkpoint_to_pytorch.py#L58

Reply:
@LysandreJik Could you review this PR? Thanks!

Reply:
No, you're looking in the wrong file. The command shown is `transformers-cli convert`, not `python convert_bert_original_tf_checkpoint_to_pytorch.py`.

If you type `transformers-cli convert` in your terminal you'll see the missing values are the ones currently set:
```
transformers-cli <command> [<args>] convert: error: the following arguments are required: --model_type, --tf_checkpoint, --pytorch_dump_output
```

Reply:
Interesting. Just a few lines above, the doc reads:

> You can convert any TensorFlow checkpoint for BERT (in particular the pre-trained models released by Google) in a PyTorch save file by using the convert_bert_original_tf_checkpoint_to_pytorch.py script.
>
> This CLI takes as input a TensorFlow checkpoint...

According to what you have said, `transformers-cli` is different from `convert_bert_original_tf_checkpoint_to_pytorch.py`. Is there a gap that I'm missing?